kvmagent/kvmagent/plugins/hygon_device_plugin.py

```
@@ -558,8 +567,106 @@ def generate_hygon_mdev_devices(self, req):
+        if r == 124:
+            logger.warning("start_qemu.py --update_qemu_conf timeout after 60s")
+        elif r != 0:
+            logger.warning("failed to execute start_qemu.py --update_qemu_conf: %s" % e)
+        else:
+            logger.info("start_qemu.py --update_qemu_conf executed successfully")
+
+        # Append /dev/hygon_psp_config to cgroup_device_acl in qemu.conf
+        self._append_hygon_psp_config_to_qemu_conf()
+
+        # Restart libvirtd service to apply qemu.conf changes
+        r, o, e = bash_roe("timeout 60 systemctl restart libvirtd")
```

Ruff (0.14.8)

Unpacked variable `o` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)